typescript/package.json

```diff
@@ -22,10 +22,11 @@
         "validate": "npm run typecheck && npm run lint",
         "storybook": "storybook dev -p 6006",
         "build-storybook": "storybook build",
-        "storybook:test": "test-storybook",
-        "docker:build": "docker build --tag wsc_story_test - < Dockerfile.test",
-        "docker:storybook:test": "docker run -it --rm --mount type=bind,source=.,target=/test --network host --ipc=host wsc_story_test",
-        "docker:storybook:test:update": "npm run docker:storybook:test -- npm run storybook:test -- --updateSnapshot",
+        "storybook:test": "test-storybook --no-index-json",
+        "docker:build": "docker build --tag wsc_story_test --file Dockerfile.test .",
+        "docker:storybook:test": "cross-env-shell CWD=$PWD npm run docker:storybook:test:docker:core -- -it --rm --mount type=bind,source=${CWD}/packages,target=/test/packages --network host --ipc=host wsc_story_test",
```

Comment:
Does using `${PWD}` directly work?

Reply:
No, it does not work

Reply:
Committed a rename from `CWD` to `PWD` for compatibility with Linux shell. Does it work for you?